201-web-app-ase-create/azuredeploy.json

```diff
@@ -0,0 +1,172 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "aseName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the App Service Environment"
+            }
+        },
+        "aseLocation": {
```

Comment:
There isn't any requirement that the location of an ASE match the default location of the resource group it is contained in.